src/components/layout/nav/search/Search.jsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState } from "react";
+import Col from "react-bootstrap/Col";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+import "./Search.scss";
+
+//This will be serving as a temporary database until we build the real one completely.
```

Comment:
as mentioned on the other change i think since omar is also using this you guys should move this out into a common file and one person PRs it

Reply:
Yes, I'm gonna `import` it from Omar's if he merged it.

Reply:
please do a separate small PR either you or omar do it, small changes are better, it should be a quick approval with the new file. in a common file.